src/python/dxpy/scripts/dx.py

```diff
@@ -6002,6 +6003,21 @@ def positive_integer(value):
 parser_unarchive.set_defaults(func=archive, request_mode="unarchival")
 register_parser(parser_unarchive, categories='fs')
 
+#####################################
+# extract_dataset
+#####################################
+parser_extract_dataset = subparsers.add_parser('extract_dataset', help='(Preview feature) Retrieves the data, or generates SQL to retrieve the data, from a dataset or cohort for a set of entity.fields.',
+                                   description='(Preview feature) Retrieves the data, or generates SQL to retrieve the data, from a dataset or cohort for a set of entity.fields.',
+                                   prog='dx extract_dataset')
+parser_extract_dataset.add_argument('path', help='v3.0 Dataset or Cohort object ID (project-id:record-id where ":record-id" indicates the current selected project) or name')
+parser_extract_dataset.add_argument('-ddd', '--dump-dataset-dictionary', action="store_true", default=False, help='If provided, the three dictionary files: <record_name>.data_dictionary.csv, <record_name>.entity_dictionary.csv, <record_name>.codings.csv are generated using the -delimiter specified, if applicable, in the output directory, if not provided, in the current location')
```

Comment:
in the current location on the Platform

Reply:
Not on the platform. The files are created on local where the command is run from. Agreed, this should be made more clear. We can change it to -> In the current location where `dx extract_dataset` is being run.

Reply:
Updated!